packages/pipes_sh.rb

```diff
@@ -0,0 +1,15 @@
+require 'package'
+
+class Pipes_sh < Package
+  description 'Animated pipes terminal screensaver'
+  homepage 'https://pipeseroni.github.io/'
+  version '1.3.0'
+  source_url 'https://github.com/pipeseroni/pipes.sh/archive/v1.3.0.tar.gz'
+  source_sha256 '532976dd8dc2d98330c45a8bcb6d7dc19e0b0e30bba8872dcce352361655a426'
+
+  depends_on 'ncurses'
+
+  def self.install
+    system "make", "PREFIX=#{CREW_DEST_DIR}#{CREW_PREFIX}", "install"
```

Comment:
You can use `#{CREW_DEST_PREFIX}` instead of the two combined constants.

Reply:
With that said, it might be better to use `DESTDIR=#{CREW_DEST_DIR}` and the normal `PREFIX=#{CREW_PREFIX}`, if possible.

Reply:
So `make DESTDIR=#{CREW_DEST_DIR} PREFIX=#{CREW_PREFIX} install` for both packages?

Reply:
If you can swing it, sure.